src/components/select/select.js

```diff
@@ -1276,7 +1276,7 @@ function SelectProvider($$interimElementProvider) {
      * Calculate the
      */
     function calculateMenuPositions(scope, element, opts) {
-      var 
+      var
```

Comment:
Ah, sorry, I see now! I originally had the style code above/below this line which is why I separated it, but I can indeed move those back together now.